site/blog/drafts/why-i-hate-markdown.rst

```diff
@@ -110,9 +110,15 @@ you can write:
 
 	.. warning:: This is a Warning!
 
+Or, in AsciiDoc, you can write:
```

Comment:
Agreed. It was the only place I could find an rST that had an obvious
asciidoc counterpart. Insufficient criterion, but my brain wouldn't go any
further :-(

On Mon, Mar 14, 2016 at 7:23 PM, Eric Holscher <notifications@github.com>
wrote:

> In site/blog/drafts/why-i-hate-markdown.rst
> <https://github.com/ericholscher/ericholscher.com/pull/3#discussion_r56093272>
> :
>
> > @@ -110,9 +110,15 @@ you can write:
> >
> >  	.. warning:: This is a Warning!
> >
> > +Or, in AsciiDoc, you can write:
>
> This example doesn't add anything :) Would be good to ahve an Adoc example
> somewhere else, so that it's more mixed between rst & adoc, not just
> "here's how to do the same thing in both" :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ericholscher/ericholscher.com/pull/3/files#r56093272>.
>